plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/jmx/JdbcClientStats.java

```diff
@@ -237,9 +237,9 @@ public JdbcApiStats getGetSplits()
 
     @Managed
     @Nested
-    public JdbcApiStats getGetTableHandle()
+    public JdbcApiStats getGetTableHandleOfTableName()
```

Comment:
Let's keep the current name for backwards compatibility of metrics and JMX reporting.

Let's pick a different name for the new method: `getTableHandleForQuery` (for stats only, not in JdbcClient interface)